docs/edgeql/functions/string.rst

```diff
@@ -4,30 +4,135 @@
 String
 ======
 
-.. eql:function:: std::lower(string: str) -> str
+.. eql:function:: std::str_lower(string: str) -> str
 
-    Return a lowercase copy of the input string.
+    Return a lowercase copy of the input *string*.
 
     .. code-block:: edgeql-repl
 
-        db> SELECT lower('Some Fancy Title');
+        db> SELECT str_lower('Some Fancy Title');
         {'some fancy title'}
 
-.. eql:function:: std::str_to_json(string: str) -> json
+.. eql:function:: std::str_upper(string: str) -> str
 
-    :index: json parse loads
+    Return an uppercase copy of the input *string*.
 
-    Return JSON value represented by the input string.
+    .. code-block:: edgeql-repl
+
+        db> SELECT str_upper('Some Fancy Title');
+        {'SOME FANCY TITLE'}
+
+.. eql:function:: std::str_title(string: str) -> str
 
-    This is the reverse of :eql:func:`json_to_str`.
+    Return a titlecase copy of the input *string*.
+
+    Every word in the *string* will have the first letter capitalized
+    and the rest converted to lowercase.
 
     .. code-block:: edgeql-repl
 
-        db> SELECT str_to_json('[1, "foo", null]');
-        {[1, 'foo', None]}
+        db> SELECT str_title('sOmE fAnCy TiTlE');
+        {'Some Fancy Title'}
+
+.. eql:function:: std::str_lpad(string: str, n: int64, fill: str = ' ') -> str
```

Comment:
`rtrim`, `ltrim` and `trim` have a similar argument, which doesn't really make as much sense to force to be NAMED ONLY. It would seem to be a bit awkward to remember which one is and isn't NAMED ONLY if it's only the `rpad` and `lpad`.